### PR TITLE
fix(integration): error message in LinkedIn Insight Tag

### PR DESCRIPTION
**Pull request by sauravlal-rudderstack**
## Description of the change

> fixing the error message

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

- - -
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/656)

### CODE REVIEW

**Review by ItsSudip — DISMISSED** [review]

File: integrations/LinkedInInsightTag/browser.js [hunk 1]
```diff
@@ -51,8 +51,8 @@ class LinkedInInsightTag {
       false
     );
     if (!eventMapping[trimmedEvent]) {
-      logger.error(
-        `Event ${trimmedEvent} doesn't matches with LinkedIn Insight Tag!!===`
+      logger.debug(
+        `Event ${trimmedEvent} doesn't matches with LinkedIn Insight Tag. Aborting!===`
```
Comment (saikumarrs) [inline integrations/LinkedInInsightTag/browser.js:55]:
```suggestion
      logger.debug(
        `The "${event}" event is not mapped in the destination dashboard. It'll be skipped`
```

File: integrations/LinkedInInsightTag/browser.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ class LinkedInInsightTag {
     const { event } = message;
     if (!event) {
       logger.error(
-        "[LinkedIn Insight Tag]: Event name is missing for track call!!==="
+        "[LinkedIn Insight Tag]: Event name is missing for track call."
```
Comment (saikumarrs) [inline integrations/LinkedInInsightTag/browser.js:42]:
```suggestion
        "[LinkedIn Insight Tag]: Event name is missing"
```

**Review by saikumarrs — CHANGES_REQUESTED** [review]

File: integrations/LinkedInInsightTag/browser.js (hunk 1 above)
Comment (sauravlal-rudderstack) [reply integrations/LinkedInInsightTag/browser.js]: Updated

File: integrations/LinkedInInsightTag/browser.js (hunk 2 above)
Comment (sauravlal-rudderstack) [reply integrations/LinkedInInsightTag/browser.js]: Updated

Comment (saikumarrs) [reply integrations/LinkedInInsightTag/browser.js]: Change my suggestion. Please incorporate it.

Comment (sauravlal-rudderstack) [reply integrations/LinkedInInsightTag/browser.js:42]: Done

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**